orbit/cmd/desktop/desktop.go

```diff
@@ -33,6 +86,11 @@ func main() {
 	}
 	log.Info().Msgf("fleet-desktop version=%s", version)
 
+	commChannelID := os.Getenv("FLEET_DESKTOP_CHANNEL_ID")
+	if commChannelID == "" {
+		log.Fatal().Msg("missing Channel ID environment FLEET_DESKTOP_CHANNEL_ID")
```

Comment:
If this is hardcoded then let's not pass it as env var (I see `[2]string{"FLEET_DESKTOP_CHANNEL_ID", constant.DesktopAppExecName}` on the other end) and have both processes assume the hardcoded name. 

This is to avoid issues when the app auto-updates Fleet Desktop first.

Reply:
good suggestion. I initially used a GUID instead of the constant, but then I thought about making it a bit more generic and just passing the hardcoded constant and keeping the env variable mechanism. The `FLEET_DESKTOP_CHANNEL_ID` env variable is part of that previous mechanism. I followed your suggestion and just left the hardcoded constant.